core/domain/suggestion_services.py

```diff
@@ -107,6 +108,11 @@ def create_suggestion(
         suggestion_type, target_type, target_id,
         target_version_at_submission, status, author_id,
         None, change, score_category, thread_id, suggestion.language_code)
+
+    # Update the community contribution stats so that the number of suggestions
+    # of this type that are in review increases by one.
+    _update_community_contribution_stats_due_to_suggestion(suggestion, 1)
```

Comment:
Better to just say what this is doing, so, e.g. _update_suggestion_counts_in_community_contribution_stats